wp-content/themes/thinkery/config/theme-supports.php

```diff
@@ -31,7 +31,7 @@
 		'skip-links',
 	],
 	'genesis-after-entry-widget-area' => '',
-	'genesis-footer-widgets'          => 3,
+	'genesis-footer-widgets'          => 5,
 	'genesis-menus'                   => [
 		'primary'   => __( 'Header Menu', 'genesis-sample' ),
 		'secondary' => __( 'Footer Menu', 'genesis-sample' ),
```

Comment:
Since we already have a footer menu location setup, we'll might want to remove it if we stay with the widget route for the footer menu. My local had a footer placeholder menu set up already, if used, it appears below the site credits. 

![Screen Shot 2019-10-29 at 10 19 56 AM](https://user-images.githubusercontent.com/26469099/67781669-bba6eb00-fa35-11e9-93b1-6097f36b8150.png)

Reply:
Good catch, i completely forgot about that item

We can remove lines 149, 150, and 151 from the functions.php file of the Child theme to remove this

```
// Repositions the secondary navigation menu.
remove_action( 'genesis_after_header', 'genesis_do_subnav' );
add_action( 'genesis_footer', 'genesis_do_subnav', 10 );
```

I am removing these from my local build currently

Reply:
LGTM!